.github/workflows/build-images-beta.yaml

```diff
@@ -67,7 +67,7 @@ jobs:
         id: tag-in-repositories
         shell: bash
         run: |
-          if docker buildx imagetools inspect quay.io/${{ github.repository_owner }}/${{ matrix.name }}-${{ github.event.inputs.suffix }}:${{ github.event.inputs.tag }} &>/dev/null; then
+          if docker buildx imagetools inspect quay.io/${{ secrets.QUAY_ORGANIZATION }}/${{ matrix.name }}-${{ github.event.inputs.suffix }}:${{ github.event.inputs.tag }} &>/dev/null; then
```

Comment:
:thinking: should beta builds be `secrets.QUAY_ORGANIZATION_DEV`?

Reply:
hmm i'm not sure about this one 🤔 do we even need a separate `cilium-beta` repository, or can we just push to `cilium` repo with some `beta` tag? for now i'll change this to `secrets.QUAY_ORGANIZATION_DEV`.

Reply:
I think that the reason this uses `quay.io/cilium/cilium-beta` is that the `quay.io/cilium/cilium-ci` can technically be written by anyone with any code, so there is no supply chain security there. `quay.io/cilium/cilium-beta` on the other hand requires a committer to approve the build, so they will likely take a look at the PR and validate that there's nothing too :fish: going on.

Reply:
At the same time, if it's a beta, then putting it in the same quay.io/cilium/cilium repo as the stable builds seems maybe not quite right somehow. But then, we put RC's there :shrug: